Makefile.ide

```diff
@@ -159,6 +159,18 @@ $(IDETOPBYTE): ide/idetop.ml $(LINKCMO) $(LIBCOQRUN) $(IDETOPCMA)
 			 $(SYSMOD) -package camlp5.gramlib \
 			 $(LINKCMO) $(IDETOPCMA) $(BYTEFLAGS) $< -o $@
 
+ifeq ($(ARCH),Darwin)
+KEYSRC:=ide/config_files/MacOS/default_accel_map
+else
+ifeq ($(ARCH),win32)
+KEYSRC:=ide/config_files/Win32/default_accel_map
+else
+KEYSRC:=ide/config_files/Other/default_accel_map
+endif
+endif
+ide/coqide.keys: $(KEYSRC)
+	cp $< $@
```

Comment:
Here's a makefile using commit. Should do the same thing as the configure code. WDYT?

Reply:
`ARCH` is put by configure in config/Makefile btw.

Reply:
> `ARCH` is put by configure in config/Makefile btw.

`make` has always `OS` in the env, right?